html/index.html

```diff
@@ -75,7 +75,7 @@
             <li><a href="./categoryPageWoman.html">WOMEN</a></li>
             <li><a href="./categoryPageMan.html">MEN</a></li>
             <li class="jeanbtn"><a href="./categoryPageJeans.html">JEANS</a></li>
-            <li><a href="">MAGAZINE</a></li>
+            <li><a href="./categoryPageAccessories.html">ACCESSORIES</a></li>
```

Comment:
should it not be called just `accessories`

Reply:
<img width="1101" alt="Screenshot 2021-07-27 at 10 44 40" src="https://user-images.githubusercontent.com/10894967/127133429-db32a5b0-33c0-43a3-8a99-34ee91b2d362.png">
 
Upper case makes sense as that is how they are all shown in the rendered HTML

Reply:
I meant the url